espnet2/asr_transducer/initialize.py

```diff
@@ -0,0 +1,75 @@
+"""Initialize ESPnet2 Transducer model modules."""
```

Comment:
What is the difference between the initialization of Transducer and that of the other models (i.e., `torch_utils/initialize.py`)? This one does not support `espnet_initialization_fn`?

Reply:
Thanks for raising this issue! This file is basically useless right now because 1) initialization is `None` in most experiments and we don't support `RNN` blocks now nor "special" init for Transducer.

I kept it because we really need to revisit initialization and I also need to adapt it for custom module. Do you want me to remove it and initialization option (or raise an issue if it's not `None`)?

Reply:
Thanks for the answer! Yeah, I think it's reasonable to keep it since we may need customized initialization in the future. For now, it might be better to raise an issue if it's not `None`, otherwise some users can get confused.

Reply:
Kept the file but I added a comment at the top + removed call and import of `initialize` in model init

Reply:
In fact, I'll remove the initialization part for now! I'm not too fan of keeping dead code even if it should be temporary.